package.json

```diff
@@ -69,7 +71,8 @@
     "reselect": "^4.0.0",
     "squarelink": "^1.1.3",
     "web3": "1.2.4",
-    "web3connect": "^1.0.0-beta.23"
+    "web3connect": "^1.0.0-beta.23",
+    "react-ga": "latest"
```

Comment:
why latest and not specific version number?

Reply:
Nothing special, now I'd changed it to `^2.7.0`